controllers/apps/v2alpha1/generated.go

```diff
@@ -136,7 +142,7 @@ func generateDashboardService(instance *appsv2alpha1.EMQX) *corev1.Service {
 			Namespace:   instance.Namespace,
 			Name:        instance.Spec.DashboardServiceTemplate.Name,
 			Labels:      instance.Spec.DashboardServiceTemplate.Labels,
-			Annotations: instance.Spec.DashboardServiceTemplate.Annotations,
+			Annotations: mergeMap(instance.Spec.DashboardServiceTemplate.Annotations, instanceAnnotations),
```

Comment:
Sorry, my bad, I think this merge should in default web hook

Reply:
> Sorry, my bad, I think this merge should in default web hook

done